cves/kernel/CVE-2019-12379.yml

```diff
@@ -55,7 +55,12 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  A possible memory leak in the Linux drivers. This issue can lead to a situation where the 
```

Comment:
The transition in wording from this line to the description of the vulnerability could be better